modules/core/keeper/msg_server_test.go

```diff
@@ -900,12 +919,11 @@ func (suite *KeeperTestSuite) TestChannelUpgradeInit() {
 					path.EndpointA.Chain.SenderAccount.String(),
 				)
 			},
-			func(res *channeltypes.MsgChannelUpgradeInitResponse, err error) {
+			func(_ sdk.Context, res *channeltypes.MsgChannelUpgradeInitResponse, err error) {
```

Comment:
one thing I'd be in favor of is doing:

```
suite.Require().Empty(ctx.EventManager().Events().ToABCIEvents())
```

instead of discarding ctx here. Can't harm to assert no-event emission when we don't expect it.

Reply:
good call! ill add that